.github/workflows/fork-sync.yml

```diff
@@ -0,0 +1,20 @@
+name: Sync Fork
+
+on:
+  schedule:
+    - cron: '0 * * * *' # every hour
+  workflow_dispatch: # on button click
+
+jobs:
+  sync:
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: tgymnich/fork-sync@v1.8
+        with:
+          owner: ethereum-optimism
+          repo: optimism
+          base: op-espresso-integration # our default branch
```

Comment:
This is the name of the fork, not our default branch. Either this value or the comment should be changed.